mojo-data/mojos/rulefit/cars/v1.00/rulefit_v1.00_cars_multiparams.txt

```diff
@@ -0,0 +1,46 @@
+2
+2
+2
+2
+2
+2
+java.lang.ArrayIndexOutOfBoundsException
```

Comment:
caused by 
 mojoModel.getNumClasses(-1) //java.lang.ArrayIndexOutOfBoundsException: -1
 mojoModel.getDomainValues(-1) //java.lang.ArrayIndexOutOfBoundsException: -1
so this is ok (-1 is invalid colId)

Reply:
the same in other outputs ending by _multiparams.txt